tests/mocks/course_mocks.py

```diff
@@ -1,17 +1,12 @@
-mocked_subject_data = {"subjects": [{"subject": "CS", "descr": "Computer Science"}]}
-
```

Comment:
These lines were accidentally deleted when you cleaned up this merge conflict, but these lines aren't part of the merge conflict